src/main/java/frc/robot/Feedback/LED/Limelight.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.Feedback.LED;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Limelight extends CommandBase {
+  /** Creates a new Limelight. */
+  private TowerSubsystem m_tower;
+  public Limelight(TowerSubsystem m_tower) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    public static NetworkTableInstance table = NetworkTableInstance.getDefault();
+    this.m_tower = m_tower;
+    addRequirements(m_tower);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (!m_tower.m_entryBeamBreak) {
+      table.getTable("limelight").getEntry("ledMode").setNumber(2);
+      //There is probably a better way to do this
+      thread.sleep(2000);
```

Comment:
Using `Thread.sleep()` here will halt all running robot code since everything on an FRC user program runs in one thread. My suggestion would be to just set the ledMode to 2 in `initialize()`, and then call this command with a timeout parameter, then when it exits, set the ledMode back to 0 in `end()`. 

https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj/command/Command.html

This javadoc article shows a bit about how to call a command with a `timeout` parameter.